packages/types/src/lib/interfaces.ts

```diff
@@ -193,7 +193,14 @@ export interface LitNodeClientConfig {
   alertWhenUnauthorized?: boolean;
   minNodeCount?: number;
   debug?: boolean;
+  // Timeout for connecting/handshaking with all nodes
   connectTimeout?: number;
+  // Timeout for connection request made to each node. If not provided, the default value is 7500ms
+  nodeConnectionTimeout?: number;
+  // Timeout for each request made to a node
+  nodeRequestTimeout?: number;
+  // Whether to retry handshaking a particular node if it failed, will retry until connectTimeout is reached
+  retryNodeHandshake?: boolean;
```

Comment:
use jsDoc

```suggestion
  /**
   * Timeout for connecting/handshaking with all nodes
   */
  connectTimeout?: number;
  /**
   * Timeout for connection request made to each node. If not provided, the default value is 7500ms
   */
  nodeConnectionTimeout?: number;
  /**
   * Timeout for each request made to a node
   */
  nodeRequestTimeout?: number;
  /**
   * Whether to retry handshaking a particular node if it failed, will retry until connectTimeout is reached
   */
  retryNodeHandshake?: boolean;
```

<img width="519" alt="image" src="https://github.com/user-attachments/assets/fd390beb-00ae-4fa8-94ac-116ddfec4002">